### PR TITLE
Remove legacy Helm charts under install/ folder

### PR DESCRIPTION
**Pull request by howardjohn**
:wave:

### CODE REVIEW

**Comment by howardjohn** [comment]
need to fix gen and release

**Comment by howardjohn** [comment]
For https://github.com/istio/istio/issues/20441

**Comment by howardjohn** [comment]
Ah, missed one issue in the release builder

File: Makefile.core.mk [hunk 1]
```diff
@@ -336,9 +336,7 @@ lint-helm-global:
 	find manifests -name 'Chart.yaml' -print0 | ${XARGS} -L 1 dirname | xargs -r helm lint --strict -f manifests/global.yaml
 
 lint: lint-python lint-copyright-banner lint-scripts lint-go lint-dockerfiles lint-markdown lint-yaml lint-licenses lint-helm-global
-	@bin/check_helm.sh
 	@bin/check_samples.sh
-	@bin/check_dashboards.sh
```
Comment (costinm) [inline Makefile.core.mk]: Do we have an equivalent with the new charts ?

File: Makefile.core.mk [hunk 2]
```diff
@@ -336,9 +336,7 @@ lint-helm-global:
 	find manifests -name 'Chart.yaml' -print0 | ${XARGS} -L 1 dirname | xargs -r helm lint --strict -f manifests/global.yaml
 
 lint: lint-python lint-copyright-banner lint-scripts lint-go lint-dockerfiles lint-markdown lint-yaml lint-licenses lint-helm-global
-	@bin/check_helm.sh
```
Comment (costinm) [inline Makefile.core.mk]: Same - we should still run a check_helm, but with the new templates.

Can be separate PR.

**Review by costinm — COMMENTED** [review]
I see you also deleted consul, intentional ? 

Also some of the install/tools.

File: Makefile.core.mk (hunk 1 above)
Comment (howardjohn) [reply Makefile.core.mk]: To the best of my knowledge this is some legacy thing that no longer applies

File: Makefile.core.mk (hunk 2 above)
Comment (howardjohn) [reply Makefile.core.mk]: We do the same as `lint-helm-global`

**Comment by howardjohn** [comment]
> I see you also deleted consul, intentional ?
> 
> Also some of the install/tools.

These are not documented so I assumed they were not used/supported, but for the sake of reducing the scope of this PR ill add them back here

**Comment by howardjohn** [comment]
@costinm I added consul and tools back as well as the GCP readme. This PR was originally meant to see what breaks then adopted it into a real PR so I didn't clean things up quite right.

**Review by costinm — APPROVED** [review]

**Review by ericvn — APPROVED** [review]

**Review by fpesce — APPROVED** [review]
LGTM
